Basic-Scripts/day_of_date.py

```diff
@@ -0,0 +1,38 @@
+#Function to calculate day of the week
+def dayofweek(
```

Comment:
Include Docstrings for function definition and follow naming conventions for functions.